src/main/java/picard/vcf/MergeVcfs.java

```diff
@@ -54,14 +54,74 @@
 import java.util.List;
 
 /**
- * Combines multiple VCF files into a single file. Input files must be sorted by their contigs
- * and, within contigs, by start position. Throws IllegalArgumentException if the contig lists
- * are not present in the input files, are not identical or if the sample lists are not the
- * same; this class uses the GATK to merge headers, which may throw exceptions if the headers
- * cannot be merged. See VCFUtils.smartMergeHeaders for details.
- * <p/>
- * An index file is created for the output file by default. Using an output file name with a
- * ".gz" extension will create gzip-compressed output.
+ * Combines multiple variant files into a single variant file.
+ *
```

Comment:
The explanations that have to do with how you can either pass in one or more input files or a list, how an index file is created by default but that can be disabled, etc, should go here rather than in example descriptions. The idea is that you explain the rules here, and then later all you need to do is show succinctly how to apply the major options.